docs/tbdev_getting_started.ipynb

```diff
@@ -1,235 +1,272 @@
 {
```

Comment:
<p>The newline formatting disappeared from this section when viewed in Colab.</p><p> </p><p>I suggest rewording the new part to: "The <code>--one_shot</code>  flag causes the uploader to exit when it has uploaded the entire <code>logdir</code> . This should be used when the experiment has already completed. Without this flag, the uploader will keep running until it is stopped, in order to read new data from the directory during ongoing training."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>